src/pages/find-buy/index.md

```diff
@@ -4,15 +4,15 @@ templateKey: page
 description: |
     Find and buy Honest Biscuits at our Pike Place Market bakery where where we bake our biscuits fresh daily. Or find us at a Seattle farmer's market, coffee shop, or at the local grocery. You can also order biscuits for delivery or ship them nationwide via mail order. We can also cater your next event or office party.
 ---
-<b>Open for Take-out and Delivery!</b> </br>
+<h1>Open for <a href={config.userLinks[4].url}>Takeout</a> and <a href={config.userLinks[6].url}>Delivery</a>!</h1>
```

Comment:
This may not work correctly as the `config.js` won't import using JS via Markdown during the build.

So it's probably better to use the URL directly for the href.